go.mod

```diff
@@ -170,7 +170,7 @@ require (
 	github.com/gogo/protobuf v1.3.2 // indirect
 	github.com/golang/protobuf v1.5.4 // indirect
 	github.com/google/btree v1.1.3 // indirect
-	github.com/google/go-cmp v0.6.0 // indirect
+	github.com/google/go-cmp v0.6.0
```

Comment:
Newer 0.7.0 is also available https://github.com/google/go-cmp/releases/tag/v0.7.0

Reply:
Probably worth bumping to the latest, while you're at it 😬

Reply:
This is a funny one... I actually didn't change it manually. 
It comes from a `go mod tidy`. and it's not a bump, it's just the `//indirect` comment that was removed.

Reply:
Funny one indeed 🤔 In that case IMO just leave it as it :)